app/assets/javascripts/discourse/app/templates/mobile/modal/create-account.hbs

```diff
@@ -0,0 +1,177 @@
+{{#create-account email=accountEmail disabled=submitDisabled action=(action "createAccount")}}
```

Comment:
Is it necessary to add a new template for mobile? Sometimes it's necessary, but if it isn't, it would be great to only have one template.

Reply:
While I def. agree one template would be better...

As far as I can tell, it is necessary because of the way the new modal is being styled. To work correctly in all browsers, the new heading needs to be positioned in a different position on mobile as opposed to the desktop version.

Reply:
got this done :+1:

Reply:
Awesome, love to see it!